.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+---
+# https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+orbs:
+  hugo: circleci/hugo@1.2.2
+
+workflows:
+  main:
+    jobs:
+      - hugo/build:
+          asciidoc: true
+          html-proofer: true
+          htmlproofer-http-status-ignore: "'0,999'"
+          htmlproofer-url-ignore: "'/coach/'"
+          htmlproofer-timeframe: "'6w'"
+          version: "latest"
+      - html-tidy:
+          filters:
+            branches:
+              only: main
+          requires:
+            - hugo/build
+      - deploy:
+          filters:
+            branches:
+              only: main
+          requires:
+            - hugo/build
+            - html-tidy
+
+jobs:
+  html-tidy:
+    docker:
+      - image: cimg/ruby:3.0
+    steps:
+      - run: gem install htmlbeautifier
```

Comment:
I would suggest that instead of *tidying* the HTML, we may want to *minify* it. Hugo has the option to minify the output. 

> Note that you can also minify the final HTML output to /public by running `hugo --minify`
> https://gohugo.io/hugo-pipes/minification/